packages/grub/grub.spec

```diff
@@ -14,7 +14,7 @@ Release: 1%{?dist}
 Summary: Bootloader with support for Linux and more
 License: GPL-3.0-or-later AND Unicode-DFS-2015
 URL: https://www.gnu.org/software/grub/
-Source0: https://cdn.amazonlinux.com/blobstore/a2f920abd554c7ab22af43c720198abcf5f78828c0543a0d7c65c654610eab26/grub2-2.06-2.amzn2.0.1.src.rpm
+Source0: https://cdn.amazonlinux.com/blobstore/21d0df3b06c1c5cc9e5cf3bb559dad713335e782ac3a46b57c5d0097e22c0aec/grub2-2.06-9.amzn2.0.1.src.rpm
```

Comment:
Can you make sure we still get the expected console output for all of the platform and arch combinations?

I'd suggest setting a non-zero timeout in `grub.cfg` and then verifying that we can see the menu for x86 and arm64 architectures on EC2 / ESXi / metal.

Reply:
I confirmed that something like this shows in the grub menu for different variants:

```
      Bottlerocket OS 1.10.0
```